llm/data_gemma/llms.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""LLM Interface."""
+
+import json
+import logging
+import time
+from typing import Any
+
+import requests
+
+from data_gemma import base
+
+
+_REQ_DATA = {
+    'contents': [{
+        'parts': [{
+            'text': '',
+        }],
+    }],
+    'generationConfig': {
+        'temperature': 0.1,
+    },
+    'safetySettings': [
+        {'category': 'HARM_CATEGORY_HARASSMENT', 'threshold': 'BLOCK_NONE'},
+        {'category': 'HARM_CATEGORY_HATE_SPEECH', 'threshold': 'BLOCK_NONE'},
+        {
+            'category': 'HARM_CATEGORY_SEXUALLY_EXPLICIT',
+            'threshold': 'BLOCK_NONE',
+        },
+        {
+            'category': 'HARM_CATEGORY_DANGEROUS_CONTENT',
+            'threshold': 'BLOCK_NONE',
+        },
+    ],
+}
+
+
+class GoogleAIStudio(base.LLM):
```

Comment:
why not put base.LLM here?

Also should file be "llm.py"?

Reply:
I renamed this file to be google api specific.